src/coreclr/vm/excep.cpp

```diff
@@ -4767,7 +4767,8 @@ LONG InternalUnhandledExceptionFilter_Worker(
 #endif // !TARGET_UNIX
 
             // Send notifications to the AppDomains.
-            NotifyAppDomainsOfUnhandledException(pParam->pExceptionInfo, NULL, useLastThrownObject, fIsProcessTerminating /*isTerminating*/);
+            // UNITY: We'll assume for now that if an event is sent that the unhandled exception has been handled until the proposed API: https://github.com/dotnet/runtime/issues/101560 has been implemented
```

Comment:
Do we want to control this via an exported function or some other mechanism?

Reply:
We could... I figure if we wanted the exception to be fatal we could always terminate from the handler?